aws/build

```diff
@@ -40,6 +40,10 @@ def main()
 
     HipChat.log "https://github.com/code-dot-org/code-dot-org/commit/#{`git rev-parse HEAD`}", message_format: 'text', color: 'purple'
 
+    # Run `sudo chef-client` to ensure that the deploy respect updates to the Chef config
+    # (e.g. the cdo-servers attribute describing which servers to deploy).
+    RakeUtils.system_ 'sudo', 'chef-client'
```

Comment:
should we use `system` (no underscores) instead so that the output goes somewhere on failure?

https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/rake_utils.rb#L27